web/forms.py

```
@@ -1734,6 +1750,17 @@ def save(self, commit=True):
         return meme
 
 
+# Form used during updates, image may not be changed by users
+class MemeEditForm(MemeForm):
+    class Meta(MemeForm.Meta):
+        fields = [f for f in MemeForm.Meta.fields if f != "image"]
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)